cmake/Kokkos.cmake

```diff
@@ -11,18 +11,26 @@ get_property(IS_EKAT_KOKKOS_BUILT GLOBAL PROPERTY EKAT_KOKKOS_BUILT SET)
 if (NOT IS_EKAT_KOKKOS_BUILT)
 
   if (NOT Kokkos_SOURCE_DIR)
-    message (FATAL_ERROR "Error! Please, specify path to Kokkos in Kokkos_SOURCE_DIR.\n")
+    message (STATUS "Kokkos_SOURCE_DIR not specified: using submodule version.")
+    if(DEFINED Kokkos_ENABLE_DEPRECATED_CODE AND Kokkos_ENABLE_DEPRECATED_CODE)
```

Comment:
I don't think you need the first part inside the if statement. `if (MY_VAR)` is automatically false if `MY_VAR` is not defined.

Reply:
Ah, right. I had the opposite logic before, so that's left over.